<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DGIpydrOne</class>
 <widget class="QMainWindow" name="DGIpydrOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DGIpydrOne</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 1px solid rgb(40, 94, 142);
	border-radius: 4px;
	background-color: rgb(66, 139, 202);
	color: rgb(255, 255, 255);
 }
 
 QPushButton:pressed {
	background-color: rgb(50, 118, 177);
 }
 
 QPushButton:flat {
	border: none;
 }
 
 QPushButton:default {
	border-color: rgb(159, 159, 159);
 }

QLineEdit {
	border: 1px solid rgb(204, 204, 204);
	border-radius: 4px;
	padding: 10px 10px 10px 10px;
	font: 11pt &quot;MS Shell Dlg 2&quot;;
	selection-background-color: darkgray;
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Commands</string>
     </attribute>
     <widget class="QDial" name="degreesDial">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSlider" name="throttleSlider">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>121</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="connectBeforeWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(190, 190, 190, 45%);
}</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1px solid rgb(188, 232, 241);
	border-radius: 4px;
	background-color: rgb(217, 237, 247);
	color: rgb(49, 112, 143);
	font: 11pt \&quot;MS Shell Dlg 2\&quot;;
}</string>
       </property>
       <property name="text">
        <string>Information : please connect to the drone.</string>
       </property>
      </widget>
     </widget>
     <widget class="QGraphicsView" name="joystickView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>256</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="joystickXInformation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Joystick X : --°</string>
      </property>
     </widget>
     <widget class="QLabel" name="joystickYInformation">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Joystick Y : --°</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkLED">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Led</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkCollision">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Collision detect</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkLanding">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Landing</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="droneInformation">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkAutoPilot">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto Pilot</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>100</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="sonarView">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>141</y>
        <width>271</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QProgressBar" name="batteryLevelBar">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>61</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="manualMode">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Manual</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="automaticMode">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatic</string>
      </property>
     </widget>
     <widget class="QSlider" name="leftRightCalibrate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-10</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="frontBackCalibrate">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>22</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-10</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <zorder>line_2</zorder>
     <zorder>line</zorder>
     <zorder>degreesDial</zorder>
     <zorder>throttleSlider</zorder>
     <zorder>joystickView</zorder>
     <zorder>joystickXInformation</zorder>
     <zorder>joystickYInformation</zorder>
     <zorder>connectBeforeWidget</zorder>
     <zorder>checkLED</zorder>
     <zorder>checkCollision</zorder>
     <zorder>checkLanding</zorder>
     <zorder>droneInformation</zorder>
     <zorder>checkAutoPilot</zorder>
     <zorder>spinBox</zorder>
     <zorder>spinBox_2</zorder>
     <zorder>spinBox_3</zorder>
     <zorder>sonarView</zorder>
     <zorder>batteryLevelBar</zorder>
     <zorder>manualMode</zorder>
     <zorder>automaticMode</zorder>
     <zorder>leftRightCalibrate</zorder>
     <zorder>frontBackCalibrate</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max power</string>
      </property>
     </widget>
     <widget class="QSlider" name="maximalPowerSliver">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sensibility</string>
      </property>
     </widget>
     <widget class="QSlider" name="sensibilitySlider">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Max angle</string>
      </property>
     </widget>
     <widget class="QSlider" name="maximalAngleSlider">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSaveProfile">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save profile</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Console/Connection</string>
     </attribute>
     <widget class="QTextEdit" name="console">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>581</width>
        <height>261</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="commandText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>329</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSendCommand">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverIp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.95.20</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonConnect">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Connection</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="serverPort">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>5005</number>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonCancelConnect">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: rgb(255, 82, 82);
}

QPushButton:pressed {
	background-color: rgb(138, 48, 48);
}</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLabel" name="connectionTime">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0s</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Remote</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_DGI_PROJECT"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuProfile"/>
   <addaction name="menuConnect"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAbout_DGI_PROJECT">
   <property name="text">
    <string>About DGI PROJECT</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
